DS/DS8/SortingSystem.java

```diff
@@ -0,0 +1,92 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+public class SortingSystem {
+	static int[] list=null;
+	public static void input() {
+		Scanner sc=new Scanner(System.in);
+		int sizeOfList=0;
+		boolean isPositive=false;
+		try {
+			while(!isPositive) {
+				System.out.println("Enter size of array");
+				sizeOfList=sc.nextInt();
+				if(sizeOfList<=0) {
+					System.out.println("Enter only positive value");
+				}
+				else {
+					isPositive=true;
+				}
+			}
+		list=new int[sizeOfList];
+		System.out.println("Enter number in list:");
+		TreeSorting.addElement(list,sc);
+		}catch(InputMismatchException exception) {System.out.println("Enter only integer value");System.exit(0);}
+	}
+	public static void output() {
```

Comment:
ACTION: Output function should just print the resultant sorted array but this function is doing lot more than that. Actual logic should not be inside output function it should inside some other function. and main function should call that function